index.js

```diff
@@ -0,0 +1,32 @@
+import {
+  createConnection,
+  ProposedFeatures,
+  TextDocuments
+} from 'vscode-languageserver/node.js'
+import {TextDocument} from 'vscode-languageserver-textdocument'
+import {configureUnifiedLanguageServer} from './server.js'
+
+/**
+ * @typedef {Pick<
+ *   import('unified-engine').Options,
+ *   'ignoreName' | 'packageField' | 'pluginPrefix' | 'plugins' | 'rcName'
+ * > & {
+ *   defaultSource?: string
+ * }} Options
+ */
+
+/**
+ * Create a language server for a unified ecosystem.
+ *
+ * @param {Options} options Configuration for `unified-engine` and the language
+ * server.
```

Comment:
```suggestion
 * @param {Options} options
 *   Configuration for `unified-engine` and the language server.
```

I’ve been moving towards this style, I find it easier to gloss over code and see what the params / descriptions are, and that the line break helps with diffs